tests/e2e/e2e_test.go

```diff
@@ -81,6 +81,8 @@ func (s *IntegrationTestSuite) TestGov() {
 	s.GovCancelSoftwareUpgrade()
 	s.GovCommunityPoolSpend()
 	s.AddRemoveConsumerChain()
+
+	s.testSetBlocksPerEpoch()
```

Comment:
This new test practically tests that governance proposals work for `BlocksPerEpoch` and hence why `testSetBlocksPerEpoch()` appears under `TestGov`.